src/components/Label.svelte

```diff
@@ -61,6 +61,7 @@
 
   .label {
     padding: 0;
+    margin: 0.25em;
```

Comment:
```suggestion
    margin: 0.125em;
```
I find it looks better to keep the tags closer in this case, but no strong opinion about this.

Reply:
Yep, agreed. I'll change this.